spyder-custom-layout/setup.cfg

```diff
@@ -0,0 +1,8 @@
+# ----------------------------------------------------------------------------
```

Comment:
Please remove this file because it's only required to create wheels for Python 2 and 3, and we don't support Python 2 anymore.